platform/server-config/src/main/java/org/motechproject/server/config/SettingsFacade.java

```diff
@@ -214,6 +220,36 @@ public void saveRawConfig(String filename, String jsonText) {
         }
     }
 
+    public String propertiesToJson(Properties props) {
```

Comment:
This looks like some logic that should be done behind the scenes for the user. The API should simply expose a method that accepts Properties and we should handle it ourselves by parsing them to json.